ee/clickhouse/views/events.py

```diff
@@ -109,10 +112,17 @@ def list(self, request: Request, *args: Any, **kwargs: Any) -> Response:
         return Response({"next": next_url, "results": result})
 
     def retrieve(self, request: Request, pk: Optional[int] = None, *args: Any, **kwargs: Any) -> Response:
```

Comment:
Q: isn't the pk type annotation completely wrong here? Isn't it supposed to be a string?

Reply:
Outside of the PR but yes, it's a string in CH and an int in PG